gtda/images/filtrations.py

```diff
@@ -349,3 +350,135 @@ def transform(self, X, y=None):
         Xt = np.concatenate(Xt)
 
         return Xt
+
+
+@adapt_fit_transform_docs
+class DilationFiltration(BaseEstimator, TransformerMixin):
+    """Filtrations of 2D/3D binary images based on distances to a reference
```

Comment:
Having a one-liner description which is identical between two different transformers seems a bad idea to me.

Reply:
In any case this one-linear does not seem to match the longer description below.